src/main/java/com/adyen/util/HMACValidator.java

```diff
@@ -79,16 +79,35 @@ public boolean validateHMAC(NotificationRequestItem notificationRequestItem, Str
         return MessageDigest.isEqual(merchantSign, expectedSign);
     }
 
-    public String getDataToSign(NotificationRequestItem notificationRequestItem) {
```

Comment:
calculateHMAC is checking for notificationRequestItem null but NPE will occur at #73 if notificationRequestItem is null.

Reply:
Fixed